Makefile

```diff
@@ -9,6 +9,7 @@ include pkgconfig.mk	# package version
 
 LIBNAME = unicorn
 UNAME_S := $(shell uname -s)
+SMP_MFLAGS := -j4
```

Comment:
could you put in some comments for this line, so users can understand what it is, and how to use it?